src/smartcontracts/argSerializer.ts

```diff
@@ -43,7 +43,7 @@ export class ArgSerializer {
      */
     stringToBuffers(joinedString: string): Buffer[] {
         // We also keep the zero-length buffers (they could encode missing options, Option<T>).
-        return joinedString.split(ArgumentsSeparator).map(item => Buffer.from(item, "hex"));
+        return joinedString.split(ARGUMENTS_SEPARATOR).map(item => Buffer.from(item, "hex"));
```

Comment:
Could be argued that moving the constant `ArgumentsSeparator` is a breaking change. If so, it's a very _small_ one. In practice, it shouldn't be at all a breaking change, though.